R/expr__expr.R

```diff
@@ -3675,16 +3675,18 @@ Expr_reshape = function(dims) {
 #' @param seed numeric value of 0 to 2^52
 #' Seed for the random number generator. If set to Null (default), a random
 #' seed value integerish value between 0 and 10000 is picked
+#' @param fixed_seed Boolean, If TRUE, The seed will not be incremented between draws.
+#' This can make output predictable because draw ordering can change due to threads being
+#' scheduled in a different order.
 #' @return  Expr
 #' @aliases shuffle
 #' @format NULL
 #' @keywords Expr
 #' @examples
 #' pl$DataFrame(a = 1:3)$select(pl$col("a")$shuffle(seed = 1))
-Expr_shuffle = function(seed = NULL) {
-  seed = seed %||% sample(0:10000, 1L)
-  if (!is.numeric(seed) || any(is.na(seed)) || length(seed) != 1L) pstop(err = "seed must be non NA/NaN numeric scalar")
-  unwrap(.pr$Expr$shuffle(self, seed))
+#' stop("new param + reworked to robj_to - > update tests of shufle")
```

Comment:
this shouldn't be in the docs, can you put it as a simple comment with "TODO"? (I didn't review the rest, I just noticed this)

Reply:
When bumping rust I just fix all the compiler errors at once and not update the R side + docs when changed behavior. This stop() was my personal TODO that prevents merging before removed again. When ready for review they will all be gone I promise ;)

Reply:
Yeah I just shouldn't review draft PR ;)